ddtrace/internal/telemetry/data.py

```diff
@@ -58,6 +60,20 @@ def _get_application(key):
     }
 
 
+def get_dependencies():
+    # type: () -> List[Dict[str, str]]
+    """Returns a unique list of the names and versions of all installed packages"""
+    deps_added = set()
+    dependencies = []
+    for dist in get_distributions():
+        dependency = (dist.name, dist.version)
+        if dependency not in deps_added:
+            dependencies.append({"name": dependency[0], "version": dependency[1]})
+        deps_added.add(dependency)
+
+    return dependencies
```

Comment:
Wouldn't this be enough? I suppose it we end up with the same name + version, but with two different paths then you could end up with the same name/version combination in the list which is not good?

```suggestion
    return [
        {"name": dist.name, "version": dist.version}
        for dist in get_distributions()
    ]
```

Reply:
Yup I was avoiding adding duplicate names and versions. I can refactor this code so it's easier to follow